sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -307,9 +308,81 @@ private static int ParseHealthcareEntityIndex(string reference)
 
         #region Analyze Operation
 
-        internal static AnalyzeOperationResult ConvertToAnalyzeOperationResult(AnalyzeJobState jobState, IDictionary<string, int> map)
+        internal static PiiTask ConvertToPiiTask(RecognizePiiEntitiesOptions option)
         {
-            return new AnalyzeOperationResult(jobState, map);
+            return new PiiTask()
+            {
+                Parameters = new PiiTaskParameters()
+                {
+                    Domain = option.DomainFilter.GetString(),
+                    ModelVersion = !string.IsNullOrEmpty(option.ModelVersion) ? option.ModelVersion : "latest",
+                    StringIndexType = option.StringIndexType
```

Comment:
do we need to set a default value for this option here or is it already set somewhere else?

Reply:
We need to set it here as the currently analyze requires the "latest" version in body.

Reply:
Also it feels weird if we set a default value for this param somewhere else and then set the default value for model version param here.

Reply:
@suhas92 I was referring to the string index type param.

Reply:
Updated and created an issue for tracking.
https://github.com/Azure/azure-sdk-for-net/issues/18542